src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/BasicStage.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.digital.ho.hocs.casework.domain.model;
+
+import lombok.NoArgsConstructor;
+import uk.gov.digital.ho.hocs.casework.domain.exception.ApplicationExceptions;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import static uk.gov.digital.ho.hocs.casework.application.LogEvent.STAGE_CREATE_FAILURE;
+
+@NoArgsConstructor
+@Entity
+@Table(name = "stage")
+public class BasicStage extends BaseStage {
```

Comment:
`BasicStage` vs `BaseStage` is confusing. Why not just `Stage`?

Reply:
I was going for the minimum diff for the PR, happy to rename the current 'Stage' to 'StageData' to reflect that it returns the data blob and 'BasicStage' to 'Stage' ? That would be more correct?

Reply:
Or `StageComposite` would suggest there's more in it
`StageWithCase` - is explicit and is without confusion..

`BasicStage` to `Stage` is correct.